C++/Logger/sinks/Sink.cpp

```diff
@@ -0,0 +1,12 @@
+#include "Sink.h"
+
+namespace Toolbox::Log::Sinks
+{
+
+Sink & Sink::operator << (const std::string & message)
+{
+  this->output (message);
+  return *this;
```

Comment:
I think it is good practice to keep the * and & next to the object when you are doing dereference and address-of.  When it is a pointer declaration, I like the space.